### ISSUE TITLE
Cache loading can crash

### ISSUE THREAD

**Issue by olegsu** [telepresenceio/telepresence#612]
### Automatically included informatio

Command line: `['/usr/local/bin/telepresence', '--run-shell']`
Version: `0.85`
Python version: `3.6.5 (default, Mar 30 2018, 06:41:53) 
[GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.39.2)]`
kubectl version: `Client Version: v1.10.1 // Server Version: v1.8.7-gke.1`
oc version: `(error: [Errno 2] No such file or directory: 'oc': 'oc')`
OS: `Darwin Oleg.local 17.3.0 Darwin Kernel Version 17.3.0: Thu Nov  9 18:09:22 PST 2017; root:xnu-4570.31.3~1/RELEASE_X86_64 x86_64`
Traceback:

```
Traceback (most recent call last):
  File "/usr/local/Cellar/telepresence/0.85/libexec/lib/python3.6/site-packages/telepresence/cli.py", line 74, in call_f
    return f(*args, **kwargs)
  File "/usr/local/Cellar/telepresence/0.85/libexec/lib/python3.6/site-packages/telepresence/main.py", line 433, in go
    runner = Runner.open(args.logfile, kubectl_or_oc(server), args.verbose)
  File "/usr/local/Cellar/telepresence/0.85/libexec/lib/python3.6/site-packages/telepresence/runner.py", line 138, in open
    open(logfile_path, "a", buffering=1), kubectl_cmd, verbose
  File "/usr/local/Cellar/telepresence/0.85/libexec/lib/python3.6/site-packages/telepresence/runner.py", line 114, in __init__
    self.cache = Cache.load(os.path.join(cache_dir, "cache.json"))
  File "/usr/local/Cellar/telepresence/0.85/libexec/lib/python3.6/site-packages/telepresence/cache.py", line 29, in load
    cache = json.load(f)
  File "/usr/local/opt/python/Frameworks/Python.framework/Versions/3.6/lib/python3.6/json/__init__.py", line 299, in load
    parse_constant=parse_constant, object_pairs_hook=object_pairs_hook, **kw)
  File "/usr/local/opt/python/Frameworks/Python.framework/Versions/3.6/lib/python3.6/json/__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "/usr/local/opt/python/Frameworks/Python.framework/Versions/3.6/lib/python3.6/json/decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/local/opt/python/Frameworks/Python.framework/Versions/3.6/lib/python3.6/json/decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

```

Logs:

```
Not available
```

**Comment by olegsu** [comment]
Just clean the cache. 
Just a suggestion, show where the cache is been stored in --help command, not only in code :-D

**Comment by ark3** [comment]
Should be fixed by #716
